sos/report/plugins/ceph_mgr.py

```diff
@@ -103,17 +72,67 @@ def setup(self):
             "version"
         ]
 
+        directory = ''
+        if not microceph_pkg:
+            directory = '/var/run/ceph'
+            self.add_file_tags({
+                '/var/log/ceph/(.*/)?ceph-mgr.*.log': 'ceph_mgr_log',
+            })
+
+            self.add_forbidden_path([
+                "/etc/ceph/*keyring*",
+                "/var/lib/ceph/**/*keyring*",
+                "/var/lib/ceph/**/osd*",
+                "/var/lib/ceph/**/mon*",
+                # Excludes temporary ceph-osd mount location like
+                # /var/lib/ceph/tmp/mnt.XXXX from sos collection.
+                "/var/lib/ceph/**/tmp/*mnt*",
+                "/etc/ceph/*bindpass*",
+            ])
+
+            self.add_copy_spec([
+                "/var/log/ceph/**/ceph-mgr*.log",
+                "/var/lib/ceph/**/mgr*",
+                "/var/lib/ceph/**/bootstrap-mgr/",
+                "/run/ceph/**/ceph-mgr*",
+            ])
+
+        else:
+            directory = '/var/snap/microceph'
+            self.add_file_tags({
+                '/var/snap/microceph/common/logs/ceph-mgr.*.log':
+                'ceph_mgr_log',
+            })
+
+            self.add_forbidden_path([
+                "/var/snap/microceph/common/**/*keyring*",
+            ])
+
+            self.add_copy_spec([
+                "/var/snap/microceph/common/logs/ceph-mgr*.log",
+            ])
+
+        self.add_cmd_output(
+            [f"ceph {cmd}" for cmd in ceph_mgr_cmds])
+
+        # get ceph_cmds again as json for easier automation parsing
+        self.add_cmd_output(
+            [f"ceph {cmd} --format json-pretty" for cmd in ceph_mgr_cmds],
+            subdir="json_output",
+        )
+
         self.add_cmd_output([
-            f"ceph daemon {m} {cmd}" for m in self.get_socks() for cmd in cmds]
+            f"ceph daemon {m} {cmd}" for m in self.get_socks(
+                directory) for cmd in cmds]
```

Comment:
Same as below, minor nit but I think it would read easier if the line-breaking included the full `self.get_socks` call on L126, or breaks after it, e.g.:

```python
self.add_cmd_output([
    f"ceph daemon {m} {cmd}" for m in self.get_socks(directory)
    for cmd in cmds
])

Reply:
done